src/SearchByTitleForm/SearchByTitleForm.js

```diff
@@ -52,6 +60,7 @@ class SearchByNameForm extends Component {
                         disabled={!this.state.movieTitle}
                         />
                 </form>
+                {this.state.movieID && this.state.submit ? <Redirect push to={'/' + this.state.movieID} /> : null}
```

Comment:
This terinary controls the Redirect. If there is a value in `movieID` and `submit` is true, this Redirects with a push into the history stack. The push just adds to the history instead of replacing the current position. This enables the back button.

Reply:
I think this is a really cool work around for not being allowed to use hooks and I'm here for it!